<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Operações / Informações</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QSplitter" name="splitter_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QToolBox" name="toolBox">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="page">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>291</width>
               <height>516</height>
              </rect>
             </property>
             <attribute name="label">
              <string>Agência</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QGroupBox" name="groupBox_3">
                <property name="title">
                 <string>Operações</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QPushButton" name="imprimir_agencia">
                   <property name="text">
                    <string>Tabela_Completa</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="folha_pagamento">
                   <property name="text">
                    <string>Folha de Pagamento</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="qtd_funcionarios">
                   <property name="text">
                    <string>Quantidade de Funcionários</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="qtd_carros">
                   <property name="text">
                    <string>Quantidade de Carros</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_4">
                <property name="title">
                 <string>Lucro de Vendas</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="QPushButton" name="total_vendas">
                   <property name="text">
                    <string>Total_Vendas</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="vendas_2015">
                   <property name="text">
                    <string>Desde de 2015</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="vendas_2010">
                   <property name="text">
                    <string>Desde de 2010</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="vendas_2005">
                   <property name="text">
                    <string>Desde de 2005</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_5">
                <property name="title">
                 <string>Lucro de Trocas</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item>
                  <widget class="QPushButton" name="total_trocas">
                   <property name="text">
                    <string>Total_Trocas</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="trocas_2015">
                   <property name="text">
                    <string>Desde 2015</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="trocas_2010">
                   <property name="text">
                    <string>Desde 2010</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="trocas_2005">
                   <property name="text">
                    <string>Desde 2005</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>291</width>
               <height>348</height>
              </rect>
             </property>
             <attribute name="label">
              <string>Carros</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                 <string>Operações</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <widget class="QPushButton" name="imprimir_carros">
                   <property name="text">
                    <string>Tabela_Completa</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="nao_vendidos">
                   <property name="text">
                    <string>Não_Vendidos</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="mais_vendido">
                   <property name="text">
                    <string>Mais_Vendidos</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="menos_vendidos">
                   <property name="text">
                    <string>Menos_Vendidos</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QGroupBox" name="groupBox">
                <property name="title">
                 <string>Mais vendidos em períodos de 5 anos</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_4">
                 <item row="1" column="0">
                  <widget class="QPushButton" name="carros_vendas_2010_2015">
                   <property name="text">
                    <string>2010 - 2015</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QPushButton" name="carros_vendas_2005_2010">
                   <property name="text">
                    <string>2005 - 2010</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QPushButton" name="carros_vendas_2015_2023">
                   <property name="text">
                    <string>2015 - 2023</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QPushButton" name="carros_vendas_2000_2005">
                   <property name="text">
                    <string>2000 - 2005</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_8">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>305</width>
               <height>239</height>
              </rect>
             </property>
             <attribute name="label">
              <string>Peças</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QGroupBox" name="groupBox_16">
                <property name="title">
                 <string>Operações</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_13">
                 <item>
                  <widget class="QPushButton" name="imprimir_pecas">
                   <property name="text">
                    <string>Tabela_Completa</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_3">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>305</width>
               <height>239</height>
              </rect>
             </property>
             <attribute name="label">
              <string>Funcionários</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QGroupBox" name="groupBox_10">
                <property name="title">
                 <string>Operações</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_9">
                 <item row="0" column="0">
                  <widget class="QPushButton" name="imprimir_funcionarios">
                   <property name="text">
                    <string>Tabela_Completa</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_11">
                <property name="title">
                 <string>Eficiência</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_9">
                 <item>
                  <widget class="QPushButton" name="mais_eficientes">
                   <property name="text">
                    <string>Mais Eficientes</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="menos_eficientes">
                   <property name="text">
                    <string>Menos Eficientes</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_4">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>291</width>
               <height>436</height>
              </rect>
             </property>
             <attribute name="label">
              <string>Clientes</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QGroupBox" name="groupBox_6">
                <property name="title">
                 <string>Operações</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_5">
                 <item row="0" column="0">
                  <widget class="QPushButton" name="imprimir_clientes">
                   <property name="text">
                    <string>Tabela_Completa</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QPushButton" name="revisao_atrasada">
                   <property name="text">
                    <string>Revisão Atrasada</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QPushButton" name="carros_antigo">
                   <property name="text">
                    <string>Carros Velhos (10 anos)</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_7">
                <property name="title">
                 <string>Compras</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_6">
                 <item row="0" column="0">
                  <widget class="QPushButton" name="maior_quantidade_compras">
                   <property name="text">
                    <string>Quantidade por Cliente</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QPushButton" name="gasto_compras">
                   <property name="text">
                    <string>Gasto por Cliente</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_8">
                <property name="title">
                 <string>Trocas</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_7">
                 <item row="1" column="0">
                  <widget class="QPushButton" name="maior_quantidade_trocas">
                   <property name="text">
                    <string>Quantidade por Cliente</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QPushButton" name="gasto_trocas">
                   <property name="text">
                    <string>Gasto por Cliente</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_9">
                <property name="title">
                 <string>Manutenção</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_8">
                 <item row="0" column="0">
                  <widget class="QPushButton" name="gasto_manutencao">
                   <property name="text">
                    <string>Gasto por Cliente</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_5">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>305</width>
               <height>239</height>
              </rect>
             </property>
             <attribute name="label">
              <string>Vendas</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_17">
              <item>
               <widget class="QGroupBox" name="groupBox_12">
                <property name="title">
                 <string>Operações</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_10">
                 <item>
                  <widget class="QPushButton" name="imprimir_vendas">
                   <property name="text">
                    <string>Tabela_Completa</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_6">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>305</width>
               <height>239</height>
              </rect>
             </property>
             <attribute name="label">
              <string>Trocas</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_18">
              <item>
               <widget class="QGroupBox" name="groupBox_13">
                <property name="title">
                 <string>Operações</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_11">
                 <item>
                  <widget class="QPushButton" name="imprimir_trocas">
                   <property name="text">
                    <string>Tabela_Completa</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_7">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>305</width>
               <height>239</height>
              </rect>
             </property>
             <attribute name="label">
              <string>Revisão</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_19">
              <item>
               <widget class="QGroupBox" name="groupBox_14">
                <property name="title">
                 <string>Operações</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_10">
                 <item row="0" column="0">
                  <widget class="QPushButton" name="imprimir_revisao">
                   <property name="text">
                    <string>Tabela_Completa</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_9">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>305</width>
               <height>239</height>
              </rect>
             </property>
             <attribute name="label">
              <string>Peças_pagas</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_20">
              <item>
               <widget class="QGroupBox" name="groupBox_15">
                <property name="title">
                 <string>Operações</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_12">
                 <item>
                  <widget class="QPushButton" name="pecas_pagas">
                   <property name="text">
                    <string>Tabela_Completa</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QTableWidget" name="table"/>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>904</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
